ui/src/main/res/xml/app_restrictions.xml

```diff
@@ -9,5 +9,6 @@
         android:description="@string/disable_config_export_description"
         android:key="disable_config_export"
         android:restrictionType="bool"
+        android:singleLineTitle="false"
```

Comment:
Does this make sense here in this context? Is this attribute respected by any GUI that shows this item?

Reply:
I'm not sure because I didn't find "disable_config_export_title" location when using the app